assets/js/paystack.js

```diff
@@ -1,5 +1,6 @@
 jQuery( function( $ ) {
 
+	$('head').append('<style>.merchant-logo img, img.merchant-logo { max-height: 30px !important; width: auto; }</style>');
```

Comment:
Can this be removed? I guess this is overriding a global style applied by Paystack?